package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "IBF-system",
-  "version": "0.58.0",
+  "version": "0.58.1",
```

Comment:
@p-phung Please do not update the version manually. This is also updated on PR merge automatically.